docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -1,204 +1,311 @@
-```
-
-## Step 3/3 Access your Kubernetes cluster
-
-(!docs/pages/includes/kubernetes-access/rbac.mdx!)
-
-### View pods in your cluster
-
-List connected clusters using `tsh kube ls` and switch between
-them using `tsh kube login`:
-
-```code
-$ tsh kube ls
-
-# Kube Cluster Name Selected
-# ----------------- --------
-# cookie
-
-# kubeconfig now points to the cookie cluster
-$ tsh kube login cookie
-# Logged into kubernetes cluster "cookie". Try 'kubectl version' to test the connection.
-
-# kubectl command executed on `cookie` but is routed through the Teleport cluster.
-$ kubectl get pods
-```
-
-## Next Steps
-
-- Take a look at a [kube-agent helm chart reference](../reference/helm-reference/teleport-kube-agent.mdx) for a full list of parameters.
+## Step 1/4. Select guided or manual enrollment
+
+There are two options for enrolling a Kubernetes cluster as a resource:
+
+- You can follow the guided enrollment steps in the Teleport Web UI.
+- You run commands and edit files manually in a terminal.
+
+The guided enrollment simplifies the deployment process by pre-populating commands
+and files with required information—for example, the token used to provision the
+Kubernetes cluster—and information you specify, such as the Kubernetes namespace,
+users, and groups to grant access to.
+
+If you can't use the Teleport Web UI or want to run commands and edit file manually,
+see [Enroll Kubernetes manually](#enroll-kubernetes-manually).
+
+## Step 2/4. Follow guided enrollment
+
+To enroll a Kubernetes cluster using the Teleport Web UI:
+
+1. Type the address for your Teleport cluster in a browser.
+
+1. Sign in using your administrative account and second factor authentication.
+
+1. Click **Enroll New Resources**.
+
+1. Type all or part of **Kubernetes** in the Search field to filter the resource types 
+   displayed, then click **Kubernetes**.
+
+1. Copy the command to add the `teleport-agent` chart to your charts repository and paste 
+   it in a terminal on the target computer where Kubernetes and Helm are installed.
+
+1. Type the Teleport service namespace and the display name to use when connecting to 
+   this cluster namespace, then click **Next**.
+
+   After you click Next, Teleport generates a script to configure and enroll the 
+   Kubernetes cluster as a resource in the Teleport cluster.
+
+1. Copy the command displayed in the Teleport Web UI and run it on the server running 
+   your Kubernetes cluster.
+
+   The Teleport Web UI displays "Successfully detected your new Kubernetes cluster" as 
+   confirmation that your cluster is enrolled. When you see this message, click **Next**
+   to continue.
+
+## Step 3/4. Test Kubernetes access
+
+You can now set up access for specific Kubernetes groups and users to test access to
+the Kubernetes cluster you just enrolled.
+
+To set up and test access:
+
+1. Type a Kubernetes group names and, optionally, user names that should have access to Kubernetes
```

Comment:
```suggestion
1. Type a Kubernetes group name and, optionally, user names that should have access to Kubernetes
```

Also, are these teleport user name or Kubernetes user names? We should disambiguate.